conductr_cli/test/test_conduct_logs.py

```diff
@@ -58,9 +52,9 @@ def test_multiple_events(self):
 
         http_method.assert_called_with(self.default_url)
         self.assertEqual(
-            strip_margin("""|TIME            HOST        LOG
-                            |Today 12:54:36  10.0.1.232  [WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed.
-                            |Today 12:54:30  10.0.1.232  [WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed.
+            strip_margin("""|TIME                       HOST        LOG
+                            |2015-07-29T12:10:34+10:00  10.0.1.232  [WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed.
+                            |2015-07-29T12:10:39+10:00  10.0.1.232  [WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed.
```

Comment:
It should be locale specific and render local time. Perhaps the %X format where it renders just the locale specific time, along with a flag for rendering the date also... (`--date`) Perhaps also `-u` or `--utc` for rendering as UTC time (`-u` is used by the `date` command to do this).

https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior